app/assets/stylesheets/components/_nav-button.scss

```diff
@@ -0,0 +1,92 @@
+$avatar-width: 32px;
+$avatar-padding: 4px;
+$login-btn-color-active: #909db2;
+$login-btn-bg-primary: #4c6aa5;
+$login-btn-bg-secondary: #1c366a;
+$login-btn-border: #162b54;
+$login-btn-border-bottom: #0f1e3b;
+$login-gradient: linear-gradient($login-btn-bg-primary, $login-btn-bg-secondary);
+$login-shadow: rgba(73, 207, 239, .2);
+$login-shadow-active: rgba(8, 84, 141, .33);
+
+header.usa-site-header a.nav-sign-out {
+  background: transparent;
+  border: 1px solid transparent;
+  border-radius: 3px;
+  cursor: pointer;
+  float: right;
+  font-size: 1.4rem;
+  height: $avatar-width + (2 * $avatar-padding); // 40px
+  line-height: 1.4rem;
+  padding: 0;
+  position: relative;
+  text-align: left;
+  text-decoration: none;
+
+  &:hover,
+  &:focus {
+    background: $login-btn-bg-primary;
+    background: linear-gradient($login-btn-bg-primary, $login-btn-bg-secondary);
```

Comment:
linear-gradient isn't always compatible with browsers, so having a back-up that works is important